packages/next/src/client/app-index.tsx

```diff
@@ -169,7 +167,7 @@ function useInitialServerResponse(cacheKey: string): Promise<JSX.Element> {
         }),
       })
 
-      return res.json()
+      return (await res.json())[0]
```

Comment:
why has this changed?

Reply:
That's because in `new ActionRenderResult(JSON.stringify(await actionHandler(...)))`, if the handler returns nothing, `undefined` will be passed. And since `JSON.stringify(undefined) === undefined` the response will be broken. Hence I wrapped the result in an array.